test/unit/sourceset.test.js

```diff
@@ -565,38 +835,27 @@ QUnit[qunitFn]('sourceset', function(hooks) {
       techOrder: ['fakeFlash', 'html5']
     });
 
-    player.src(flashSrc);
-
     player.ready(function() {
-      // the first sourceset comes from our FakeFlash because it extends Html5 tech
-      // which calls load() on dispose for various reasons
-      player.one('sourceset', function(e1) {
-        // ignore the first sourceset that gets called when disposing the original tech
-
-        // the second sourceset ends up being the second source because when the first source is set
-        // the tech isn't ready so we delay it, then the second source comes and the tech is ready
-        // so it ends up being triggered immediately.
-        player.one('sourceset', function(e2) {
-          assert.equal(e2.src, sourceTwo.src, 'the second sourceset ends up being the second source');
-          sourcesets++;
-
-          // now that the tech is ready, we will re-trigger the original sourceset event
-          // and get the first source
-          player.one('sourceset', function(e3) {
-            assert.equal(e3.src, sourceOne.src, 'the third sourceset is the first source');
-            sourcesets++;
-
-            assert.equal(sourcesets, 2, 'two sourcesets');
-            player.dispose();
-            delete Tech.techs_.FakeFlash;
-            done();
-          });
-        });
+      // the second sourceset ends up being the second source because when the first source is set
```

Comment:
looks like I messed up this comment.
I believe the first sourceset will be the second source so it should be:
> the *first* sourceset ends up being the second source